app/components/BuynBTC.tsx

```diff
@@ -0,0 +1,72 @@
+import { Card, CardContent } from "./ui/card";
+import { Button } from "./ui/button";
+import { FormInput } from "./form/FormInput";
+import { FormProvider, useForm } from "react-hook-form";
+import { useContext } from "react";
+import { WalletContext } from "~/providers/ByieldWalletProvider";
+import { ByieldWallet } from "~/types";
+import { SuiModal } from "./Wallet/SuiWallet/SuiModal";
+
+interface ExchangeRateProps {
+	fee: number;
+	youReceive: number;
+}
+
+function Fee({ youReceive }: ExchangeRateProps) {
+	return (
+		<Card className="p-4 bg-azure-10 rounded-2xl h-14">
+			<CardContent className="flex flex-col justify-between p-0">
+				<div className="flex justify-between gap-2">
+					<p className="text-gray-400">You Receive</p>
+					<p>{youReceive} nBTC</p>
+				</div>
+			</CardContent>
+		</Card>
+	);
+}
+
+interface OtcBuyForm {
+	numberOfSuiCoins: number;
+}
+
+export function BuynBTC() {
+	const { connectedWallet } = useContext(WalletContext);
+	const isSuiWalletConnected = connectedWallet === ByieldWallet.SuiWallet;
+
+	// TODO: get the current nBTC price
+	const nBTCPrice = 1.2;
```

Comment:
you already have it in config - why don't you use config?

Reply:
Yes, it's removed in the next PR.